README.md

```diff
@@ -275,7 +275,9 @@ Yazılım geliştirme ile ilgili özenle seçilmiş Türkçe kaynaklar listesi.
 - [Ahmet Yasin Ak](https://www.youtube.com/channel/UC06Q0ZtwUCd2uY4-2KUpUIg/feed)
 - [Data İstanbul](https://www.youtube.com/channel/UCmIscBfFWiT1avWgSCwJIeg)
 - [NedirTV](https://www.youtube.com/user/nedirtvcom)
+- [Ekrem Karaca](https://www.youtube.com/channel/UCx8gTyAppCVGMoIOCdUZGQg)
```

Comment:
home değil de videos tabının linkini alabilir miyiz? home'da tüm feed çıkıyor. videos'da sadece upload edilmişlerin listesi çıkıyor. mesela beğendiğiniz videolar home'da çıkıyor ama videos tabında çıkmıyor. bu yüzden videos daha mantıklı olacaktır https://www.youtube.com/channel/UCx8gTyAppCVGMoIOCdUZGQg/videos